bin/run-tests.sh

```diff
@@ -13,6 +13,21 @@
 
 exit_code=0
 
+# in case we're not running in Docker:
```

Comment:
This section is for accomodating `bin/run-tests.sh`.
The example-empty-file test is expected to fail when not running in Docker because the path /opt/test-runner is hardcoded.